apptail/instance.go

```diff
@@ -37,7 +38,7 @@ func (instance *Instance) Identifier() string {
 }
 
 // Tail begins tailing the files for this instance.
-func (instance *Instance) Tail() {
+func (instance *Instance) Tail(tracker storage.Tracker) {
```

Comment:
Should `Tracker` be an attribute inside of `Instance` (since each instance would belong to one tracker)?
(I could go either way on this, just wanted to get more discussion on it)